### PR TITLE
WIP: Add 0.5.0 release blog

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**
The PR includes 0.5.0 version of meshery release blog and is a draft needs to be reformat and need more  discussion

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+date:   2021-01-25 08:00:00 +0000
+image: /assets/images/posts/2020-08-07-announcing-meshery-v040/meshery-v040.png
```
Comment (Aisuko) [inline collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md:4]: Should we use the special version of the image to meshery like `meshery 0.5.0 release`?

File: collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#### Meshery
+
+
+
+The two new adapters with the latest version of Meshery released 
+
+* Meshery Adapter for  <a href="https://github.com/layer5io/meshery-traefik-mesh">Traefik Mesh (beta)</a>
+* Meshery Adapter for <a href="https://github.com/layer5io/meshery-nginx-sm">NGINX SM Mesh (beta)</a>
+
+< Todo need more detail>
```
Comment (Aisuko) [inline collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md:32]: Need more details?

File: collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<div class="meshery-list-item"></div><br/>
+
+
+#### MeshSync
+
+* MeshSync one of a family of custom controllers within Meshery Operator.
+* We released it to a new project, welcome to contribute <a href="https://github.com/layer5io/meshsync">MeshSync</a>
+
+<div class="meshery-list-item"></div>
+
```
Comment (Aisuko) [inline collections/_blogs/2021-01-25-announcing-meshery-v0.5.0.md:54]: Should we add the other components to here?

**Review by Aisuko — COMMENTED** [review]
@leecalcote Please kindly take a look

**Comment by leecalcote** [comment]
@Aisuko thanks for this. @Tanuj22 will you point @Aisuko to the `layer5-ng` branch and blog post contributing instructions for the new site?

**Comment by Tanuj22** [comment]
@Aisuko here is the sample blog post (https://github.com/layer5io/layer5/blob/layer5-ng/src/collections/blog/blog-template/index.mdx) and blog contributing resource (https://docs.google.com/document/d/1rvUZy2_S1a2_14BAQIg6b9cMhUuu04kYzkOPDPaPptI/edit#heading=h.ww7azeqddtdl)

**Comment by Aisuko** [comment]
Close sue to #1443

**Pull request closed without merging**